website/docs/d/priority.html.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: "pagerduty"
+page_title: "PagerDuty: pagerduty_priority"
+sidebar_current: "docs-pagerduty-datasource-priority"
+description: |-
+  Get information about a priority that you can use with ruleset_rules, etc.
+---
+
+# pagerduty\_priority
+
+Use this data source to get information about a specific [priority][1] that you can use for other PagerDuty resources.
+
+## Example Usage
+
+```hcl
+data "pagerduty_priority" "p1" {
+  name = "P1"
+}
+
+resource "pagerduty_ruleset" "foo" {
+  name = "Primary Ruleset"
+}
+resource "pagerduty_ruleset_rule" "foo" {
+  ruleset = pagerduty_ruleset.foo.id
+  position = 0
+  disabled = "false"
+  conditions {
+    operator = "and"
+	subconditions {
+	  operator = "contains"
+	  parameter {
+	    value = "disk space"
+		path = "payload.summary"
+	  }
+	}
+	subconditions {
+	  operator = "contains"
+	  parameter {
+	    value = "db"
+	    path = "payload.source"
+	  }
+	}
+  }
+  actions {
+    route {
+	  value = "P5DTL0K"
+	}
+	priority {
+      value = pagerduty_priority.p1.id
```

Comment:
I guess it should be `data.pagerduty_priority.p1.id` instead of `pagerduty_priority.p1.id`

Reply:
@stmcallister Any thoughts on this?